File: LAB QUIZZES/CSE221_Quiz06/_A7E/_A7E_Testing.py
import os, sys, ast, time, random, threading, subprocess
# Compile and run: pypy _A7E_Testing.py
import heapq
def solve(N, M, Us, Vs, Ws):
    res = -1
    # your code here
    return res

# stdin
# 4 3
# 1 4 3
# 3 4 4
# 2 3 5

# N, M = map(int, input().split())
# Us, Vs, Ws = [], [], []
# for _ in range(M):
#     u, v, w = map(int, input().split())
#     Us.append(u)
#     Vs.append(v)
#     Ws.append(w)

# # hardcoded input
N, M = 4, 3
Us = [1, 3, 2]
Vs = [4, 4, 3]
Ws = [3, 4, 5]
start = time.time()
res = solve(N, M, Us, Vs, Ws)
finish = time.time()
elapsed = finish - start

# # file output
# with open("OutputOfYourCode.txt", "w") as fout:
#     fout.write(str(res))

# stdout
print(res)

print("Execution time for the targeted one test is " + f"{elapsed:.9f}" + "s", flush=True)
os._exit(0)

File: LAB QUIZZES/CSE221_Quiz06/_A7E/_A7E_Testing.java
import java.util.*;
import java.io.*;

// Compile and run: javac _A7E_Testing.java && java _A7E_Testing
public class _A7E_Testing {
    public static class Pair<K, V> {
        public K key;
        public V value;

        public Pair(K key, V value) {
            this.key = key;
            this.value = value;
        }

        public String toString() {
            return "(" + key + ", " + value + ")";
        }
    }
    public static Long solve(int N, int M, ArrayList<Integer> Us, ArrayList<Integer> Vs, ArrayList<Integer> Ws) {
        Long res = -1L;
        // Place your code here
        return res;
    }

    public static void main(String args[]) throws Exception {
        // stdin
        // 4 3
        // 1 4 3
        // 3 4 4
        // 2 3 5
        // 
        // BufferedReader fin = new BufferedReader(new InputStreamReader(System.in));
        // String[] tempA = fin.readLine().split(" ");
        // int N = Integer.parseInt(tempA[0]);
        // int M = Integer.parseInt(tempA[1]);
        //
        // ArrayList<Integer> Us = new ArrayList<>();
        // ArrayList<Integer> Vs = new ArrayList<>();
        // ArrayList<Integer> Ws = new ArrayList<>();
        // for(int i = 0; i < M; i++){
        //     tempA = fin.readLine().split(" ");
        //     int u = Integer.parseInt(tempA[0]);
        //     int v = Integer.parseInt(tempA[1]);
        //     int w = Integer.parseInt(tempA[2]);
        //     Us.add(u);
        //     Vs.add(v);
        //     Ws.add(w);
        // }
        // fin.close();


        // // hardcoded input
        int N = 4, M = 3;
        ArrayList<Integer> Us = new ArrayList<>(Arrays.asList(1, 3, 2));
        ArrayList<Integer> Vs = new ArrayList<>(Arrays.asList(4, 4, 3));
        ArrayList<Integer> Ws = new ArrayList<>(Arrays.asList(3, 4, 5));

        long start = System.nanoTime();
        Long res = solve(N, M, Us, Vs, Ws);
        long finish = System.nanoTime();
        long elapsed = finish - start;

        // // file output
        // BufferedWriter fout = new BufferedWriter(new
        // FileWriter("OutputOfYourCode.txt"));
        // fout.write(res.toString());
        // fout.close();

        // stdout
        System.out.println(res);

        System.out.printf("Execution time for the targeted one test is %.9fs\n", elapsed * 1e-9);
        System.exit(0);
    }
}
